src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature

```diff
@@ -10,10 +10,10 @@ Feature: #Enter feature name here
     Then a drop down list should appear in the "Person" row
     Then the list in the "Person" row should contain "Local Persons, ULAN Persons"
       And clicks on the Create button
-    Then the vocabulary name in the titlebar should contain "Local Person"
+    Then the titlebar should contain "Local Person"
```

Comment:
This change and the one on line 18 overwrites recent changes in https://github.com/collectionspace/cspace-cucumber/commit/04f5ab6a9cad62904b27be9cd9e8384a626727d2. Could you verify that this is intended? Thanks!